.deepsource.toml

```diff
@@ -0,0 +1,7 @@
+version = 1
+
+[[analyzers]]
+name = "go"
+
+  [analyzers.meta]
+  import_root = "github.com/duduchristian/go-gpp"
```

Comment:
Is this file supposed to be included? If so, should the `import_root` not be modified to point to this repository?